docs/content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -15,25 +15,28 @@ menu:
 
 # Upgrade from Gogs
 
-Gogs versions up to 0.9.146 (db schema version 15) can be smoothly upgraded to Gitea.
+Only ancient versions (up to 0.9.146) of Gogs can be easily migrated to Gitea.
```

Comment:
Ancient sounds like they've been found near a pyramid in egypt and they are probably from a few thousand years ago. I would suggest to place `Only gogs up to 0.9.146 can be...`